### PR TITLE
Update saap-authorization-roles.md

### PR DESCRIPTION
**Pull request by rasheedamir**

### CODE REVIEW

**Comment by stakater-user** [comment]
@rasheedamir Image is available for testing. `docker pull docker-openshift-stakater-nexus.apps.prod.clusters.stakater.io/stakater/StakaterCloudDocs:SNAPSHOT-PR-178-3afb239e`

**Comment by stakater-user** [comment]
@rasheedamir Image is available for testing. `docker pull docker-openshift-stakater-nexus.apps.prod.clusters.stakater.io/stakater/StakaterCloudDocs:SNAPSHOT-PR-178-341b3ad3`

**Comment by stakater-user** [comment]
@rasheedamir Image is available for testing. `docker pull docker-openshift-stakater-nexus.apps.prod.clusters.stakater.io/stakater/StakaterCloudDocs:SNAPSHOT-PR-178-6a02e472`

File: content/sre/authentication-authorization/saap-authorization-roles.md [hunk 1]
```diff
@@ -1,56 +1,67 @@
...
+- can view service accounts/roles/role bindings in customer owned namespaces
+- can create/view/delete on UserIdentityMappings
+- can create/verify tokens and access
+- can not view service accounts/roles/role bindings in Stakater owned namespaces 
+
 ### Administration:
-- create/edit/delete resource quotas and limits on the cluster
-- access the reserved ‘saap-customer-admin’ project on the cluster, which allows for the creation of ServiceAccounts with elevated privileges and gives the ability to update default limits and quotas for projects on the cluster
+- can create/edit/delete resource quotas and limits on the cluster
+- can access the reserved ‘saap-customer-admin’ project on the cluster, which allows for the creation of ServiceAccounts with elevated privileges and gives the ability to update default limits and quotas for projects on the cluster
```
Comment (TehreemNisa) [inline content/sre/authentication-authorization/saap-authorization-roles.md:58]: change saap-customer-admin to saap-admin namespace , we removed customer in all places in code

**Review by TehreemNisa — APPROVED** [review]

**Comment by stakater-user** [comment]
@TehreemNisa Image is available for testing. `docker pull docker-openshift-stakater-nexus.apps.prod.clusters.stakater.io/stakater/StakaterCloudDocs:v0.0.120`

**Pull request merged**